repotracker/repotracker.go

```diff
@@ -409,9 +409,11 @@ func (repoTracker *RepoTracker) GetProjectConfig(ctx context.Context, revision s
 	return project, nil
 }
 
+// AddBuildBreakSubscriptions will subscribe admins of a project to a version if no one
```

Comment:
to a task not a version right?

Reply:
Logically it's a version, it's just implemented at a task level